extensions/commands/management/cmd_clean.py

```diff
@@ -0,0 +1,53 @@
+from conan.api.conan_api import ConanAPI
+from conan.api.output import ConanOutput, Color
+from conan.cli.command import OnceArgument, conan_command
+from conans.client.userio import UserInput
+
+
+recipe_color = Color.BRIGHT_BLUE
+removed_color = Color.BRIGHT_YELLOW
+
+
+@conan_command(group="Custom commands")
+def clean(conan_api: ConanAPI, parser, *args):
```

Comment:
This needs a more specific name, there will be many commands.
Something like ``clean_old_revisions`` or the like